src/clincoded/static/components/individual_curation.js

```diff
@@ -1253,8 +1253,8 @@ function IndividualCommonDiseases() {
                 )
             }
             <Input type="textarea" ref="phenoterms" label={LabelPhenoTerms()} rows="2"
-                value={individual && individual.termsInDiagnosis ? individual.termsInDiagnosis : ''}
-                labelClassName="col-sm-5 control-label" wrapperClassName="col-sm-7" groupClassName="form-group" />
+                value={individual && individual.termsInDiagnosis ? individual.termsInDiagnosis : ''} error={this.getFormError('phenoterms')}
+                clearError={this.clrFormErrors.bind(null, 'phenoterms')} labelClassName="col-sm-5 control-label" wrapperClassName="col-sm-7" groupClassName="form-group" required={this.state.proband_selected} />
```

Comment:
One suggestion could be to say something like "required when individual is proband" by setting `<Input customErrorMsg="..." />`. That said, I don't see any issue without it. I'll go ahead approve the PR, so feel free to merge!

Reply:
I'll go ahead and add that change! I'll ping for a re-approval since the reviews will be stale after I change

Reply:
For sure! Feel free to ping on Slack --